### ISSUE TITLE
client-side chat prediction cannot be disabled

### ISSUE THREAD

**Issue by tchncs** [minetest/minetest#4976]
Hi there,
today i tried to colorize the whole chat, because players are sometimes unable to see the white messages with bright background.
Well - it works everywhere so far except the regular global messages.

After looking into the code of the rank mod where i am pretty sure it worked without any duplicates in the past, the current code looks like:

```
minetest.register_on_chat_message(function(name, message)
    if message:sub(1,1) == "/" then
        return false
    else
        minetest.chat_send_all(core.colorize("#A3B5CB", "<"..name.."> ") .. core.colorize("#66BEF5", message))
        return true
    end
        return ""
end)
```
->
![2016-12-31-232744_1920x1200_scrot](https://cloud.githubusercontent.com/assets/10698140/21579583/6f126318-cfb9-11e6-85e9-068e7f14d331.png)

Unfortunally also the rank mod has this problem now too - i am still pretty sure that it worked in the near past - not sure on which git state exactly since i drived on the arch aur git package and didn't inspected the buildnumber each time but well, now i am at b95f543d and it creates those duplicates.

**Comment by bigfoot547** [comment]
@tchncs I noticed this when I was making the mod, but the uncolored messages is not broadcast to other clients. That is why I return `true` in `on_chat_message`.
I talked about this here [here](https://forum.minetest.net/viewtopic.php?t=15772#p238510).

**Comment by red-001** [comment]
What is the propose of the chat prediction anyway? It doesn't seem necessary. Lag is hardly a problem for chat.

**Comment by red-001** [comment]
fixed by 2f56a00d9eef82052614e5854a07b39b087efd0b

**Comment by tchncs** [comment]
thanks :)
